bot/utils/extension.py

```diff
@@ -0,0 +1,32 @@
+import glob
+import importlib
+import inspect
+import typing as t
+
+
+def dotted_path() -> t.Generator:
+    """Prepares all the files that are in bot/exts."""
+    files = [
+        file
+        for file in glob.glob("bot/exts/*/*.py", recursive=True)
+        if "__" not in file
```

Comment:
```suggestion
        if "_" not in file
```
It is a single underscore.

Reply:
Some files have names such as `eval_helper` so if we only use one underscore.